azkaban-web-server/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I am afraid you won't be able to create comments inside json files. However, I just wanted to ask if this needs to be in the root directory of the subproject? 

I mean, we generally keep json / resource files in resources etc, but this seems to be a package configuration. I am not that aware about nodejs so I'll let you decide whether this needs to reside in the root level of the project. What are the best practices for this ?